front/EditProfileButton.tsx

```diff
@@ -14,7 +14,7 @@ const EditProfileButton = ({ isCurrentUser }: EditProfileButtonProps) => (
       href={routes.userEdit()}
       className="btn btn-sm btn-outline-secondary action-btn"
     >
-      <i className="ion-gear-a" /> Edit Profile Settings
+      <i data-cy='edit-settings' className="ion-gear-a" /> Edit Profile Settings
```

Comment:
It is a good practice to use double quotes for attributes in JSX. In this case, for the 'data-cy' attribute.